src/ducks/Watchlists/gql/hooks.js

```diff
@@ -1,6 +1,7 @@
-import { useState, useEffect, useMemo } from 'react'
+import React, { useState, useEffect, useMemo } from 'react'
```

Comment:
check imports order in that file again, pls

Reply:
kk